src/wp-includes/class-wp-theme-json.php

```diff
@@ -2000,6 +2009,8 @@ protected static function compute_style_properties( $styles, $settings = array()
 			}
 		}
 
+		wp_cache_set( $cache_key, $declarations, 'wp-styles', DAY_IN_SECONDS );
```

Comment:
I'm unsure about the best expiration time here. If there is a side effect that we've not accounted for in the cache key value, showing stale styles for a full day without flushing the cache is pretty long. Maybe limiting this to once an hour for now would be safer?

```suggestion
		wp_cache_set( $cache_key, $declarations, 'theme_json', HOUR_IN_SECONDS );
```

Reply:
Theme json is a none presient cache, so there is no need for an expiry here.